### DISCUSSION TITLE
Pdo_sqlsrv and sqlsrv not working for php8.1

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#17436] [General] [answered]
anyone else having some big problems with pdo_sqlsrv and sqlsrv for php8.1? I've got it on php7.4 but not working on php8.1

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
this is optional
`pdo_sqlsrvpdo_sqlsrv - Library required if integration with Wapro ERP or Comarch is active`
